README.md

```diff
@@ -25,7 +25,7 @@ cordova plugin add usabilla-cordova --save
 ```
 <preference name="UseSwiftLanguageVersion" value="4.0" />
 ```
-The Usabilla bridge is compatible with Cordova 8.  Currently Cordova 9+ is not supported.
+This version of the Usabilla native bridge / wrapper works with the latest release of `XCode 11` and `Android X`, also supports stable `Cordova v9` .
```

Comment:
I would rephrase it into

This version of the Usabilla Cordova bridge supports stable `Cordova v9`, works with the latest release of `XCode 11` and targets the use of `Android X` libraries.